ddtrace/errortracking/_handled_exceptions/bytecode_injector.py

```diff
@@ -43,15 +46,18 @@ def _inject_handled_exception_reporting(func, callback: t.Optional[CallbackType]
 
     original_code = code_to_instr.__code__  # type: CodeType
 
-    callback = callback or _default_errortracking_exc_callback
+    callback = callback or _default_bytecode_exc_callback
 
     # Find the bytecode offsets, they must be the first offset of a line as we inject
     # bytecodes only at line start.
     injection_context = InjectionContext(original_code, callback, offsets_callback)
 
     # Bytecode injection and code replacement
     code, _ = inject_invocation(injection_context, original_code.co_filename, "my.package")
-    code_to_instr.__code__ = code
+    try:
+        code_to_instr.__code__ = code
+    except Exception:
+        log.debug("Could not set the code of %s", code_to_instr, exc_info=True)
```

Comment:
can we leave a comment here explaining when we would expect an exception here to occur?

Reply:
resolved in 5d6c8b91ae4780d0c1626c8cf43339f307af5378